pydis_site/apps/api/viewsets/bot/pagination.py

```diff
@@ -0,0 +1,12 @@
+from rest_framework.pagination import LimitOffsetPagination
+from rest_framework.response import Response
+
+
+class LimitSetPagination(LimitOffsetPagination):
+    """Extend LimitOffsetPagination."""
+
+    default_limit = 100
+
+    def get_paginated_response(self, data: list) -> Response:
```

Comment:
It's technically not limited to list data. The type annotation should be `typing.Any` since it can be a wide range of different types.